terraform/deployments/vpc/iam_access_analyzers.tf

```diff
@@ -0,0 +1,260 @@
+# Creates IAM Access Analyzers in each AWS region.
+# This file has loads of providers because Terraform requires one provider per region.
+
+provider "aws" {
```

Comment:
Should we move all the provider blocks to `main.tf` where the other provider lives?